code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2343,10 +2418,34 @@
 						to_chat(H, "<b><span class='warning'>Something bursts out from inside you!</span></b>")
 						message_admins("[key_name(H)] has gibbed and spawned a new cyber horror due to nanobots. ([formatJumpTo(H)])")
 						H.visible_message("<b><span class='warning'>[H]'s body rips aparts to reveal something underneath!</b></span>")
-						new /mob/living/simple_animal/hostile/monster/cyber_horror(H.loc)
-						spawning_horror = 0
-						H.gib()
-
+						if(H.species.name == "Vox")
```

Comment:
Go to species.dm, make a variable under /datum/species something like cyber_horror type and put the typepath for the cyber horror it is supposed to spawn here.

Alternatively do what slimes do,
```
var/typepath = text2path(/mob/living/simple_animal/hostile/monster/cyber_horror/[H.species.name]")
if(ispath(typepath))
     new typepath(H.loc)
else 
    new /mob/living/simple_animal/hostile/monster/cyber_horror(H.loc)
```
(This requires all cyber horrors to use the species name in their typepath)